ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po

```diff
@@ -3009,6 +3015,9 @@ msgstr ""
 msgid "Skip to site information"
 msgstr ""
 
+msgid "Are you sure you want to delete user - {name}?"
+msgstr "Are you sure you want to delete record - {name}?"
```

Comment:
@RabiaSajjad this should be `Are you sure you want to delete user- {name}?`, not `record`. Silly babel

Reply:
or `""` in that case